tests/at_functional_test/test/info_verb_test.dart

```diff
@@ -0,0 +1,68 @@
+// ignore_for_file: unused_import
+
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:at_functional_test/conf/config_util.dart';
+import 'package:test/test.dart';
+
+import 'at_demo_data.dart';
+import 'functional_test_commons.dart';
+import 'pkam_utils.dart';
+
+void main() {
+  // ignore: unused_local_variable
+  var firstAtsign =
+      ConfigUtil.getYaml()!['first_atsign_server']['first_atsign_name'];
+  Socket? socketFirstAtsign;
+
+  setUp(() async {
+    var firstAtsignServer =
+        ConfigUtil.getYaml()!['first_atsign_server']['first_atsign_url'];
+    var firstAtsignPort =
+        ConfigUtil.getYaml()!['first_atsign_server']['first_atsign_port'];
+
+    socketFirstAtsign =
+        await secure_socket_connection(firstAtsignServer, firstAtsignPort);
+    socket_listener(socketFirstAtsign!);
+  });
+
+  test('info verb test without authentication', () async {
+    await socket_writer(socketFirstAtsign!, 'info');
+    var infoVerbResponse = await read();
+    print('info verb response : $infoVerbResponse');
+    infoVerbResponse = infoVerbResponse.replaceAll('data:', '');
+    var infoResponse = jsonDecode(infoVerbResponse);
+    expect(infoResponse['version'], isNotEmpty);
+  });
+
+  // commenting the test as the server doesn't have enroll verb changes yet
```

Comment:
Please use a check of server version to determine whether to run the test or not